src/gie.c

```diff
@@ -516,6 +517,20 @@ static int ignore (const char *args) {
     return 0;
 }
 
+static int require_grid (const char *args) {
```

Comment:
This is a great idea! Previously we've handled missing grids by adding `ignore pjd_failed_to_load_grid`. I think those would be handled better by this new `require_grid` keyword.